src/client/routes/dashboard/OrganizerDash.tsx

```diff
@@ -167,8 +167,11 @@ const barStatusOptions: ChartOptions = {
 };
 
 const pieShirtData = (data: any) => {
-	const shirtData = Object.values(data).slice(0, -1);
 	const shirtLabels = Object.keys(data).slice(0, -1);
+
+	// this casting solution is bad - we should just fix the explicit any in the method param
```

Comment:
Might be worth labeling TODO or adding an issue/making item of mega any-crushing thread

Reply:
#277